pytato/transform/parameter_study.py

```diff
@@ -0,0 +1,421 @@
+from __future__ import annotations
+
+
+"""
```

Comment:
```suggestion
__doc__ = """
```
(Won't work as a docstring if there's a statement prior.)